### DISCUSSION TITLE
[Training Log 2022-06-24] The loss starts going down today🤩

### DISCUSSION THREAD

**Discussion by zh-zheng** [OpenBMB/CPM-Live#41] [Training Logs 训练日志]
# CPM-Live Training Log (June, 24)

Time: June, 24 2022 16:00

Recorder: @zh-zheng

## Loss
- Begin: 2.73
- End: 2.70  
	
## Completed Data
- $\approx$ 288.10GB

## Average Grad Norm
- 0.85

## Progress
- 30.93%

## Comment

We are glad to see the training loss is going down now😃. It looks like our model is on the right track. CPM-Ant Go! Go! Go!
